plan9port fails to build on Linux (ArchLinux 64bit).

This is the whole install log:
```
	fontsrv dependencies found.
* Compiler version:
	gcc version 15.1.1 20250425 (GCC) 
* Building mk...
cd lib9
++ pwd
echo cd /home/bret/plan9/src/lib9
cd /home/bret/plan9/src/lib9
9c _exits.c
egrep: warning: egrep is obsolescent; using grep -E
9c _p9dialparse.c
egrep: warning: egrep is obsolescent; using grep -E
9c _p9dir.c
egrep: warning: egrep is obsolescent; using grep -E
9c announce.c
egrep: warning: egrep is obsolescent; using grep -E
9c argv0.c
egrep: warning: egrep is obsolescent; using grep -E
9c atexit.c
egrep: warning: egrep is obsolescent; using grep -E
9c atoi.c
egrep: warning: egrep is obsolescent; using grep -E
9c atol.c
egrep: warning: egrep is obsolescent; using grep -E
9c atoll.c
egrep: warning: egrep is obsolescent; using grep -E
9c atnotify.c
egrep: warning: egrep is obsolescent; using grep -E
9c await.c
egrep: warning: egrep is obsolescent; using grep -E
9c cistrcmp.c
egrep: warning: egrep is obsolescent; using grep -E
9c cistrncmp.c
egrep: warning: egrep is obsolescent; using grep -E
9c cistrstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c cleanname.c
egrep: warning: egrep is obsolescent; using grep -E
9c convD2M.c
egrep: warning: egrep is obsolescent; using grep -E
9c convM2D.c
egrep: warning: egrep is obsolescent; using grep -E
9c convM2S.c
egrep: warning: egrep is obsolescent; using grep -E
9c convS2M.c
egrep: warning: egrep is obsolescent; using grep -E
9c crypt.c
egrep: warning: egrep is obsolescent; using grep -E
9c ctime.c
egrep: warning: egrep is obsolescent; using grep -E
9c dial.c
egrep: warning: egrep is obsolescent; using grep -E
9c dirfstat.c
egrep: warning: egrep is obsolescent; using grep -E
9c dirfwstat.c
egrep: warning: egrep is obsolescent; using grep -E
9c dirmodefmt.c
egrep: warning: egrep is obsolescent; using grep -E
9c dirstat.c
egrep: warning: egrep is obsolescent; using grep -E
9c dirwstat.c
egrep: warning: egrep is obsolescent; using grep -E
9c dup.c
egrep: warning: egrep is obsolescent; using grep -E
9c encodefmt.c
egrep: warning: egrep is obsolescent; using grep -E
9c errstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c exec.c
egrep: warning: egrep is obsolescent; using grep -E
9c execl.c
egrep: warning: egrep is obsolescent; using grep -E
9c exitcode.c
egrep: warning: egrep is obsolescent; using grep -E
9c fcallfmt.c
egrep: warning: egrep is obsolescent; using grep -E
9c frand.c
egrep: warning: egrep is obsolescent; using grep -E
9c '-DPLAN9_TARGET="/home/bret/plan9"' get9root.c
egrep: warning: egrep is obsolescent; using grep -E
9c getcallerpc.c
egrep: warning: egrep is obsolescent; using grep -E
9c getenv.c
egrep: warning: egrep is obsolescent; using grep -E
9c getfields.c
egrep: warning: egrep is obsolescent; using grep -E
9c getnetconn.c
egrep: warning: egrep is obsolescent; using grep -E
9c getns.c
egrep: warning: egrep is obsolescent; using grep -E
9c getuser.c
egrep: warning: egrep is obsolescent; using grep -E
9c getwd.c
egrep: warning: egrep is obsolescent; using grep -E
9c jmp.c
egrep: warning: egrep is obsolescent; using grep -E
9c lrand.c
egrep: warning: egrep is obsolescent; using grep -E
9c lnrand.c
egrep: warning: egrep is obsolescent; using grep -E
9c main.c
egrep: warning: egrep is obsolescent; using grep -E
9c malloc.c
egrep: warning: egrep is obsolescent; using grep -E
9c malloctag.c
egrep: warning: egrep is obsolescent; using grep -E
9c mallocz.c
egrep: warning: egrep is obsolescent; using grep -E
9c nan.c
egrep: warning: egrep is obsolescent; using grep -E
9c needsrcquote.c
egrep: warning: egrep is obsolescent; using grep -E
9c needstack.c
egrep: warning: egrep is obsolescent; using grep -E
9c netcrypt.c
egrep: warning: egrep is obsolescent; using grep -E
9c netmkaddr.c
egrep: warning: egrep is obsolescent; using grep -E
9c notify.c
egrep: warning: egrep is obsolescent; using grep -E
9c nrand.c
egrep: warning: egrep is obsolescent; using grep -E
9c nulldir.c
egrep: warning: egrep is obsolescent; using grep -E
9c open.c
egrep: warning: egrep is obsolescent; using grep -E
9c opentemp.c
egrep: warning: egrep is obsolescent; using grep -E
9c pin.c
egrep: warning: egrep is obsolescent; using grep -E
9c pipe.c
egrep: warning: egrep is obsolescent; using grep -E
pipe.c:12:12: warning: argument 1 of type ‘int[2]’ with mismatched bound [-Warray-parameter=]
   12 | p9pipe(int fd[2])
      |        ~~~~^~~~~
/home/bret/plan9/include/libc.h:772:24: note: previously declared as ‘int *’
  772 | extern  int     p9pipe(int*);
      |                        ^~~~
cc1: note: unrecognized command-line option ‘-Wno-deprecated-pragma’ may have been intended to silence earlier diagnostics
9c post9p.c
egrep: warning: egrep is obsolescent; using grep -E
9c postnote.c
egrep: warning: egrep is obsolescent; using grep -E
9c qlock.c
egrep: warning: egrep is obsolescent; using grep -E
9c quote.c
egrep: warning: egrep is obsolescent; using grep -E
9c rand.c
egrep: warning: egrep is obsolescent; using grep -E
9c read9pmsg.c
egrep: warning: egrep is obsolescent; using grep -E
9c readcons.c
egrep: warning: egrep is obsolescent; using grep -E
9c readn.c
egrep: warning: egrep is obsolescent; using grep -E
9c rfork.c
egrep: warning: egrep is obsolescent; using grep -E
9c searchpath.c
egrep: warning: egrep is obsolescent; using grep -E
9c sendfd.c
egrep: warning: egrep is obsolescent; using grep -E
9c sleep.c
egrep: warning: egrep is obsolescent; using grep -E
9c strdup.c
egrep: warning: egrep is obsolescent; using grep -E
9c strecpy.c
egrep: warning: egrep is obsolescent; using grep -E
9c sysfatal.c
egrep: warning: egrep is obsolescent; using grep -E
9c syslog.c
egrep: warning: egrep is obsolescent; using grep -E
9c sysname.c
egrep: warning: egrep is obsolescent; using grep -E
9c time.c
egrep: warning: egrep is obsolescent; using grep -E
9c tm2sec.c
egrep: warning: egrep is obsolescent; using grep -E
9c tokenize.c
egrep: warning: egrep is obsolescent; using grep -E
9c truerand.c
egrep: warning: egrep is obsolescent; using grep -E
9c u16.c
egrep: warning: egrep is obsolescent; using grep -E
9c u32.c
egrep: warning: egrep is obsolescent; using grep -E
9c u64.c
egrep: warning: egrep is obsolescent; using grep -E
9c unsharp.c
egrep: warning: egrep is obsolescent; using grep -E
9c wait.c
egrep: warning: egrep is obsolescent; using grep -E
9c waitpid.c
egrep: warning: egrep is obsolescent; using grep -E
9c write.c
egrep: warning: egrep is obsolescent; using grep -E
9c zoneinfo.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/dofmt.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fltfmt.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmt.c
egrep: warning: egrep is obsolescent; using grep -E
fmt/fmt.c:55:17: error: implicit declaration of function ‘ATOMIC_VAR_INIT’; did you mean ‘ATOMIC_FLAG_INIT’? [-Wimplicit-function-declaration]
   55 |                 ATOMIC_VAR_INIT(27),
      |                 ^~~~~~~~~~~~~~~
      |                 ATOMIC_FLAG_INIT
fmt/fmt.c:55:17: error: initializer element is not constant
fmt/fmt.c:55:17: note: (near initialization for ‘fmtalloc.nfmt’)
cc1: note: unrecognized command-line option ‘-Wno-deprecated-pragma’ may have been intended to silence earlier diagnostics
9c -Ifmt fmt/fmtfd.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtfdflush.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtlocale.c
egrep: warning: egrep is obsolescent; using grep -E
9c fmtlock2.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtnull.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtquote.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtrune.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fmtvprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/fprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c frexp.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/nan64.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/print.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runefmtstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runeseprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runesmprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runesnprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runesprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runevseprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runevsmprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/runevsnprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/seprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/smprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/snprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/sprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/strtod.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/vfprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/vseprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/vsmprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/vsnprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/charstod.c
egrep: warning: egrep is obsolescent; using grep -E
9c -Ifmt fmt/pow10.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/rune.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrcat.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrchr.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrcmp.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrcpy.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrdup.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrlen.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrecpy.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrncat.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrncmp.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrncpy.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrrchr.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runestrstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/runetype.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utfecpy.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utflen.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utfnlen.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utfrrune.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utfrune.c
egrep: warning: egrep is obsolescent; using grep -E
9c utf/utfutf.c
egrep: warning: egrep is obsolescent; using grep -E
9ar rsc /home/bret/plan9/lib/lib9.a _exits.o _p9dialparse.o _p9dir.o announce.o argv0.o atexit.o atoi.o atol.o atoll.o atnotify.o await.o cistrcmp.o cistrncmp.o cistrstr.o cleanname.o convD2M.o convM2D.o convM2S.o convS2M.o crypt.o ctime.o dial.o dirfstat.o dirfwstat.o dirmodefmt.o dirstat.o dirwstat.o dup.o encodefmt.o errstr.o exec.o execl.o exitcode.o fcallfmt.o frand.o get9root.o getcallerpc.o getenv.o getfields.o getnetconn.o getns.o getuser.o getwd.o jmp.o lrand.o lnrand.o main.o malloc.o malloctag.o mallocz.o nan.o needsrcquote.o needstack.o netcrypt.o netmkaddr.o notify.o nrand.o nulldir.o open.o opentemp.o pin.o pipe.o post9p.o postnote.o qlock.o quote.o rand.o read9pmsg.o readcons.o readn.o rfork.o searchpath.o sendfd.o sleep.o strdup.o strecpy.o sysfatal.o syslog.o sysname.o time.o tm2sec.o tokenize.o truerand.o u16.o u32.o u64.o unsharp.o wait.o waitpid.o write.o zoneinfo.o dofmt.o fltfmt.o fmt.o fmtfd.o fmtfdflush.o fmtlocale.o fmtlock2.o fmtnull.o fmtprint.o fmtquote.o fmtrune.o fmtstr.o fmtvprint.o fprint.o frexp.o nan64.o print.o runefmtstr.o runeseprint.o runesmprint.o runesnprint.o runesprint.o runevseprint.o runevsmprint.o runevsnprint.o seprint.o smprint.o snprint.o sprint.o strtod.o vfprint.o vseprint.o vsmprint.o vsnprint.o charstod.o pow10.o rune.o runestrcat.o runestrchr.o runestrcmp.o runestrcpy.o runestrdup.o runestrlen.o runestrecpy.o runestrncat.o runestrncmp.o runestrncpy.o runestrrchr.o runestrstr.o runetype.o utfecpy.o utflen.o utfnlen.o utfrrune.o utfrune.o utfutf.o
ar: fmt.o: No such file or directory
cd ..
cd libbio
++ pwd
echo cd /home/bret/plan9/src/libbio
cd /home/bret/plan9/src/libbio
9c bbuffered.c
egrep: warning: egrep is obsolescent; using grep -E
9c bfildes.c
egrep: warning: egrep is obsolescent; using grep -E
9c bflush.c
egrep: warning: egrep is obsolescent; using grep -E
9c bgetc.c
egrep: warning: egrep is obsolescent; using grep -E
9c bgetrune.c
egrep: warning: egrep is obsolescent; using grep -E
9c bgetd.c
egrep: warning: egrep is obsolescent; using grep -E
9c binit.c
egrep: warning: egrep is obsolescent; using grep -E
9c boffset.c
egrep: warning: egrep is obsolescent; using grep -E
9c bprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c bputc.c
egrep: warning: egrep is obsolescent; using grep -E
9c bputrune.c
egrep: warning: egrep is obsolescent; using grep -E
9c brdline.c
egrep: warning: egrep is obsolescent; using grep -E
9c brdstr.c
egrep: warning: egrep is obsolescent; using grep -E
9c bread.c
egrep: warning: egrep is obsolescent; using grep -E
9c bseek.c
egrep: warning: egrep is obsolescent; using grep -E
9c bvprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c bwrite.c
egrep: warning: egrep is obsolescent; using grep -E
9ar rsc /home/bret/plan9/lib/libbio.a bbuffered.o bfildes.o bflush.o bgetc.o bgetrune.o bgetd.o binit.o boffset.o bprint.o bputc.o bputrune.o brdline.o brdstr.o bread.o bseek.o bvprint.o bwrite.o
cd ..
cd libregexp
++ pwd
echo cd /home/bret/plan9/src/libregexp
cd /home/bret/plan9/src/libregexp
9c regcomp.c
egrep: warning: egrep is obsolescent; using grep -E
9c regerror.c
egrep: warning: egrep is obsolescent; using grep -E
9c regexec.c
egrep: warning: egrep is obsolescent; using grep -E
9c regsub.c
egrep: warning: egrep is obsolescent; using grep -E
9c regaux.c
egrep: warning: egrep is obsolescent; using grep -E
9c rregexec.c
egrep: warning: egrep is obsolescent; using grep -E
9c rregsub.c
egrep: warning: egrep is obsolescent; using grep -E
9ar rsc /home/bret/plan9/lib/libregexp9.a regcomp.o regerror.o regexec.o regsub.o regaux.o rregexec.o rregsub.o
cd ..
cd cmd/mk
++ pwd
echo cd /home/bret/plan9/src/cmd/mk
cd /home/bret/plan9/src/cmd/mk
9c arc.c
egrep: warning: egrep is obsolescent; using grep -E
9c archive.c
egrep: warning: egrep is obsolescent; using grep -E
9c bufblock.c
egrep: warning: egrep is obsolescent; using grep -E
9c env.c
egrep: warning: egrep is obsolescent; using grep -E
9c file.c
egrep: warning: egrep is obsolescent; using grep -E
9c graph.c
egrep: warning: egrep is obsolescent; using grep -E
9c job.c
egrep: warning: egrep is obsolescent; using grep -E
9c lex.c
egrep: warning: egrep is obsolescent; using grep -E
9c main.c
egrep: warning: egrep is obsolescent; using grep -E
9c match.c
egrep: warning: egrep is obsolescent; using grep -E
9c mk.c
egrep: warning: egrep is obsolescent; using grep -E
9c parse.c
egrep: warning: egrep is obsolescent; using grep -E
9c recipe.c
egrep: warning: egrep is obsolescent; using grep -E
9c rc.c
egrep: warning: egrep is obsolescent; using grep -E
9c rule.c
egrep: warning: egrep is obsolescent; using grep -E
9c run.c
egrep: warning: egrep is obsolescent; using grep -E
9c sh.c
egrep: warning: egrep is obsolescent; using grep -E
9c shell.c
egrep: warning: egrep is obsolescent; using grep -E
9c shprint.c
egrep: warning: egrep is obsolescent; using grep -E
9c symtab.c
egrep: warning: egrep is obsolescent; using grep -E
9c var.c
egrep: warning: egrep is obsolescent; using grep -E
9c varsub.c
egrep: warning: egrep is obsolescent; using grep -E
9c word.c
egrep: warning: egrep is obsolescent; using grep -E
9c unix.c
egrep: warning: egrep is obsolescent; using grep -E
9l -o o.mk arc.o archive.o bufblock.o env.o file.o graph.o job.o lex.o main.o match.o mk.o parse.o recipe.o rc.o rule.o run.o sh.o shell.o shprint.o symtab.o var.o varsub.o word.o unix.o
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
/bin/ld: cannot find -l9: No such file or directory
collect2: error: ld returned 1 exit status
install o.mk /home/bret/plan9/bin/mk
install: cannot stat 'o.mk': No such file or directory
cd ..
* Error: mk failed to build.

```
The things I have tried but haven't worked:
Exporting $PLAN9 in my .zshrc

> export PLAN9=/home/bret/plan9/plan9port

Running the INSTALL script twice in a row but it gave me the same errors.

--- a/bin/9c
+++ b/bin/9c
@@ -18,7 +18,6 @@ usegcc()
 		-Wno-stringop-truncation \
 		-Wno-stringop-overflow \
 		-Wno-format-truncation \
-		-Wno-deprecated-pragma \
 		-Wno-unused-but-set-variable \
 		-Wno-deprecated-declarations \
 		-fno-omit-frame-pointer \

--- a/bin/9c
+++ b/bin/9c
@@ -23,6 +23,7 @@ usegcc()
 		-fno-omit-frame-pointer \
 		-fsigned-char \
 		-fcommon \
+		-std=c17 \
 	"
 	# want to put -fno-optimize-sibling-calls here but
 	# that option only works with gcc3+ it seems

--- a/bin/9c
+++ b/bin/9c
@@ -23,7 +23,7 @@ usegcc()
 		-fno-omit-frame-pointer \
 		-fsigned-char \
 		-fcommon \
-		-std=c17 \
+		-std=c11 \
 	"
 	# want to put -fno-optimize-sibling-calls here but
 	# that option only works with gcc3+ it seems

--- a/include/u.h
+++ b/include/u.h
@@ -40,7 +40,7 @@ extern "C" {
 #endif
 #define _LARGEFILE64_SOURCE 1
 #define _FILE_OFFSET_BITS 64
-
+#define __USE_POSIX 1 //needed for sigjmp_buf in glibc setjmp.h
 #include <inttypes.h>
 
 #include <unistd.h>

